Server/Controllers/MatchController.cs

```diff
@@ -258,9 +267,20 @@ public async Task<ActionResult> After([FromBody] AfterMatchRequest request)
 
             _context.AddRange(newPlayers);
             _context.Matches.Add(match);
-            await _context.SaveChangesAsync();
 
-            return Ok();
+            var playersKvp = request.Players.ToDictionary(x => x.SteamId, x => x.Team);
+            var teams = _ratingService.SplitTeams(playersKvp, request.Winner, allPlayers);
```

Comment:
`SplitTeams` can just accept `(IEnumerable<MatchPlayer> players, ushort winner)`, since `MatchPlayer` already includes players' team

Reply:
again, I did this change, although there is no gain. The method still accepts 3 param, you still need to convert it to a dictionary in order to not do a loop in a loop and you still need to send the players

Reply:
MatchPlayer does include a reference to the Player entity, it's just not initialized in this code path.